### ISSUE TITLE
Latest version is not working with sail enabled

### ISSUE THREAD

**Issue by Alph44rchitect** [open-southeners/vscode-laravel-pint#34]
**Describe the bug**
The latest extension version is not working with Laravel Sail setting enabled. It seems to be ignored.

**To Reproduce**
1. Install Laravel sail
2. Install latest version of the extension
3. Set `enableDebugLogs: true`
4. Set `runInLaravelSail: true`
5. Run format command
6. Watch the output log and see that is still using the non sail path

**Expected behavior**
Using the sail path to run pint.

**Environment:**
 - Operating system: Ubuntu 22.04 via Windows with WSL2
 - IDE / version: VSCode / latest 
 - Extension version: 1.1.4
 - Extension's config: `enableDebugLogs: true, runInLaravelSail: true`

**Additional context**
The version 0.7.3 works as expected.

**Comment by d8vjork** [comment]
Thanks for raising the issue! 

Will take a look at this

**Comment by benjamindonze** [comment]
+1 on this issue

**Comment by jseitel** [comment]
@d8vjork have you considered something like this?

docker-compose exec -T laravel.test ./vendor/bin/pint

This executes the command inside the default app container that comes with Sail and logs the output to terminal. Should be executable from the shell context of VSCode and its extensions (works fine for me locally anyway.)

**Comment by d8vjork** [comment]
@jseitel Problem with that still the same, need to find out where is that `docker-compose` (or in case of newer versions, `docker compose`), could be specified on an extension setting tho

**Comment by Alph44rchitect** [comment]
@d8vjork I haven't looked into the details but it was working with v0.7.3 and still is? What changed?

**Comment by d8vjork** [comment]
> @d8vjork I haven't looked into the details but it was working as expected with v0.7.3 and still is? What changed?

You are actually right... after checking 0.7.x I discovered that I was actually wrapping the execution into a VSCode Task, will try if I could achieve the same on this new refactor I did on v1

**Comment by ricardosoares09** [comment]
Laravel pint vscode, still isn't working with docker/sail, is there any development on this ? Is there any workaround?

**Comment by AprilOneDay** [comment]
> Laravel pint vscode, still isn't working with docker/sail, is there any development on this ? Is there any workaround?

use 0.7.3

**Comment by GabrielDSousa** [comment]
I encountered a similar issue with the latest version of the Laravel Pint VSCode extension not working correctly in a Laravel Sail setup. After some troubleshooting, I found a solution that worked for me, especially if you're working in a `.devcontainer` environment.

### My Solution:

If you're using a `.devcontainer` environment, add the following snippet to your `devcontainer.json` settings and rebuild the container:

```json
"settings": {
    "laravel-pint.executablePath": "vendor/laravel/pint/builds/pint",
    "[php]": {
        "editor.defaultFormatter": "open-southeners.laravel-pint",
        "editor.formatOnSave": true
    }
}
```

### The Problem:
It seems that the default path for Laravel Pint (`vendor/bin/pint`) has been moved in the latest Laravel Sail installation, likely to `vendor/laravel/pint/builds/pint`.

### Alternative Solution:
If you're not using a `.devcontainer` environment, you could try changing the `executablePath` in your workspace or extension config to match the new path.

I hope this helps anyone else encountering the same issue!

**Comment by csouza1995** [comment]
Just to report, I have same issue! Waiting for new version, but using version 0.7.3 as commented above

**Comment by gyulavoros** [comment]
Applying the same mapping that [better-pest](https://github.com/open-southeners/vscode-laravel-pint/issues/49#issuecomment-2439550772) does seems like a good solution to unblock the Sail integration.

Finding the executable is only half of the puzzle, but file paths need to be mapped as well.

E.g., the following doesn't work:
```
sail bin pint --config /Users/{HOME}/{PATH_TO_PROJECT}/pint.json /Users/{HOME}/{PATH_TO_PROJECT}/some.php
```
as the project folder is mapped to `/var/www/html` inside the container.

Potentially, in case the extension runs in "Sail-mode", it would invoke:
```
sail bin pint --config /var/www/html/pint.json /var/www/html/some.php
```

**Comment by crazynds** [comment]
Same problem

**Comment by d8vjork** [comment]
This is most likely fixed on latest releases

If not please feel free to open a new issue
